inline_scan.sh

```diff
@@ -136,7 +136,7 @@ get_and_validate_analyzer_options() {
             C  ) clean_flag=true;;
             V  ) V_flag=true;;
             R  ) R_flag=true; PDF_DIRECTORY="${OPTARG}";;
-            v  ) v_flag=true; VOLUME_PATH="${OPTARG}";;
+            v  ) v_flag=true; VOLUME_PATH="${OPTARG}/$(date +%s)";;
```

Comment:
added to avoid collision and permit multiple runs when specifying the same folder via `-v` flag

Reply:
also, should we do this also with the default `/tmp/sysdig` folder?

Reply:
Let's think about the meaning of `-v`.
The help above states: "Use this absolute PATH for intermediate tar files. Path will be created if not existing. Default is /tmp/sysdig (ex: -v $PWD/temp)" .

So, to simplify the script especially later on, at this point it's fine to treat both kind of paths as the same thing. I'd propose the following:
- Take the `-v` argument, or use `/tmp/sysdig` if not supplied
- Check that the supplied argument is indeed an absolute path and exit with a clear error message if not; it's way too easy to write `-v ./tmp` by accident
- Create the base directory if doesn't exist
- Inside, create a directory with a clear name like `sysdig-inline-scan-$(date +%s)` and save the absolute path to it in a variable
- At the end, if everything went well it's indeed possible to `rm -rf` the absolute path to that directory (e.g. `/absolute/path/to/sysdig-inline-scan-123456789`) so we don't have to go through the pain of keeping track of each individual file. I know I said something different last time, but it was because we were trying to `rm -rf ${VOLUME_ARG_AS_SUPPLIED}`.